code_annotations/contrib/config/openedx_events_annotations.yaml

```diff
@@ -12,6 +12,7 @@ annotations:
     - ".. event_description:":
     - ".. event_data:":
     - ".. event_key_field:":
+    - ".. event_warning:":
```

Comment:
Should the fields for events be documented somewhere? I see https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html mentioned at L9 but it doesn't contain anything about events

Reply:
I think the L9 was a copy-paste error, perhaps. And yes, this should definitely be documented somewhere. I going to find them a place in the openedx-events' docs. Thanks!

Reply:
Here's the PR with the updated docs: https://github.com/openedx/openedx-events/pull/432

Reply:
Done! Thank you both